File: website/public/javascripts/models/message.js
CBR.Models.Message = new Class({
    Extends: CBR.Models.JsonSerializableModel,

    options: {  // Defaults
    },

    getId: function () {
        return this.options.id;
    },

    getFromUserId: function () {
        return this.options.fromUserId;
    },

    getToUserId: function () {
        return this.options.toUserId;
    },

    getTitle: function () {
        return this.options.title;
    },

    setTitle: function (title) {
        this.options.title = title;
    },

    getText: function () {
        return this.options.text;
    },

    getCreationDatetime: function () {
        return this.options.creationDatetime;
    },

    getReplyToMessageId: function () {
        return this.options.replyToMessageId;
    }
});

File: selenium-tests/src/main/java/models/Message.java
package models;

public class Message {
    private String title;
    private String text;
    private User from;
    private User to;

    public Message(String title,
                   String text,
                   User from,
                   User to) {
        this.title = title;
        this.text = text;
        this.from = from;
        this.to = to;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getText() {
        return text;
    }

    public void setText(String text) {
        this.text = text;
    }

    public User getFrom() {
        return from;
    }

    public void setFrom(User from) {
        this.from = from;
    }

    public User getTo() {
        return to;
    }

    public void setTo(User to) {
        this.to = to;
    }
}
